apps/daimo-mobile/src/view/screen/AddDeviceScreen.tsx

```diff
@@ -76,6 +77,20 @@ function AddDeviceScreenInner({ account }: { account: Account }) {
   const { status, message, cost, exec } = useSendAsync({
     dollarsToSend: 0,
     sendFn,
+    pendingOp: {
+      type: "keyRotation",
+      status: OpStatus.pending,
+      slot: nextSlot,
+      rotationType: "add",
+      timestamp: Date.now() / 1e3,
```

Comment:
👍 
might be easier to do `type: "keyAdd"` / `type: "keyRemove"`? but this works too